fx-src/FirefoxStorage.js

```diff
@@ -299,6 +302,40 @@ export default class FirefoxAdapter extends BaseAdapter {
         return result[0].getResultByName("last_modified");
       });
   }
+
+  /**
+   * Reset the sync status of every record and collection we have
+   * access to.
+   */
+  resetSyncStatus() {
```

Comment:
I've been doing `collection.db.resetSyncStatus()`, because based on #567 it seemed like it wouldn't be as easy/beautiful to add something similar for the IndexedDB adapter.